### DISCUSSION TITLE
Let's Connect

### DISCUSSION THREAD

**Discussion by MrAshwin2142** [MrAshwin2142/The-Wall-of-Projects#6] [General]
Hey @JohnPatrickAncajas can we connect on LinkedIn , X, or Insta.

**Comment by JohnPatrickAncajas** [comment]
That would be a pleasure Sir @MrAshwin2142! My LinkedIn name is John Patrick Ancajas and the link is: www.linkedin.com/in/
john-patrick-ancajas-2b808828a. My X username is John Patrick Ancajas | @_PatrickAncajas and the link is https://twitter.com/_PatrickAncajas.

**Comment by MrAshwin2142** [comment]
hey @JohnPatrickAncajas we can show name of contributor and number of projects add. It looks good. I am working on it, I will improve design.  I want to know your thoughts on this update.
![Screenshot 2023-11-03 145914](https://github.com/MrAshwin2142/The-Wall-of-Projects/assets/89156541/c63f0e25-dfe7-4c93-bc8b-6b9e06b45d95)

**Reply by JohnPatrickAncajas** [reply]
Hello sir! I think it looks very good and a few more styling will make it perfect. I do think it's layout will need to change at small screen devices like by going to the bottom of the projects to fit properly.

**Reply by MrAshwin2142** [reply]
yeah .. i will do it .

**Reply by MrAshwin2142** [reply]
or we create this an new issue so any new contributor can contribute by this ..

**Reply by JohnPatrickAncajas** [reply]
It might be tricky for new people to do it but I think leaving it as an issue for a while will open some opportunities and will be better in the end.

**Reply by MrAshwin2142** [reply]
ohh them I implement logic and create issue for styling ..>

**Reply by JohnPatrickAncajas** [reply]
That's a good idea!
